### ISSUE TITLE
Clean up contributors' metadata

### ISSUE THREAD

**Issue by Antoine-Zimmermann** [HyperAgents/hmas#94]
The contributors descriptions have multiple issues.

1. There are real world entities that are identified by URIs that dereference with a 200 OK, thus making them information resources. E.g. `https://inria.fr/`, `https://www.mines-stetienne.fr/`, `https://www.unisg.ch/en`, `https://www.emse.fr/~boissier/`.
2. The property `schema:url` should relate a thing to its URL (or a URL that's about the thing), thus better use a literal with type `xsd:anyURI`, but it is currently used to relate a thing to an information resource. Sometimes, it is used to relate a thing to itself, such as the thing identified by `https://inria.fr/` is its own URL.
3. The property `schema:identifier` should relate a thing to an identifier, thus a literal, not to an information resource.
4. Affiliations (`schema:affiliation`) are not webpages or websites.
5. @DrLeturc is identified with a URL that does not dereference properly (404).

These problems make it difficult to declare the schema.org properties as either ObjectProperties or DatatypeProperties. Making them AnnotationProperties would make them pure non-logical metadata with no relevance to the knowledge described, which is probably not what we, and other people, want.

**Comment by FabienGandon** [comment]
NB: this is an extract from schema.org showing we could have used schema:url for homepages:

```
Jane's home page:
  <a href="http://www.janedoe.com" property="url">janedoe.com</a>

```
from https://schema.org/Person

but I will use `schema:mainEntityOfPage`

**Comment by FabienGandon** [comment]
Closed as implemented in PR #164
